model/qti/ImportService.php

```diff
@@ -94,7 +94,7 @@ class ImportService extends ConfigurableService
 
     public const PROPERTY_QTI_ITEM_IDENTIFIER = 'http://www.tao.lu/Ontologies/TAOItem.rdf#QtiItemIdentifier';
 
-    private const MEDIA_ROOT_CLASS = 'http://www.tao.lu/Ontologies/TAOMedia.rdf#Media';
+    public const MEDIA_ROOT_CLASS = 'http://www.tao.lu/Ontologies/TAOMedia.rdf#Media';
```

Comment:
Since you do not use it here anymore, can you move it to `SharedStimulusFactory` and leave private?

Reply:
good catch, thanks